syft/pkg/cataloger/javascript/parse_package_json.go

```diff
@@ -135,7 +134,7 @@ func licenseFromJSON(b []byte) (string, error) {
 	return "", errors.New("unable to unmarshal license field as either string or object")
 }
 
-func licensesFromJSON(p PackageJSON) ([]string, error) {
```

Comment:
Why the change from function to method here? Just preference?

Reply:
initially I was going the route of custom per-cataloger package constructors, but realized it wasn't going to work given some extra information I found. I left the per-package creation updates as is though (this was one of them).